tests/c/switch_non_default.newcg.c

```diff
@@ -9,8 +9,6 @@
 //     jitstate: stop-tracing
 //     --- Begin aot ---
 //     ...
-//     switch %{{10_1}}, bb{{bb14}}, [300 -> bb{{bb11}}, 299 -> bb{{bb12}}] [safepoint: {{safepoint_id}}, (%0_0, %0_1, %0_3, %0_4, %0_5, %0_6, %10_1)]
```

Comment:
Removing this line means the test isn't really testing anything. So *something* should go here instead, right?